manifests/params.pp

```diff
@@ -17,9 +17,9 @@
     'force_confmiss'       => false,
   }
 
-  case fact('lsbdistid') {
+  case downcase($facts['os']['name']) {
```

Comment:
Is this really the same thing? I'm not too familiar with the LSB facts and always avoided them if I could.

Reply:
yes it is:
```
root@basteles-bastelknecht ~ # puppet facts show os
{ 
  "os": {
    "architecture": "amd64",
    "distro": {
      "codename": "bullseye",
      "description": "Debian GNU/Linux 11 (bullseye)",
      "id": "Debian",
      "release": {
        "full": "11.3",
        "major": "11",
        "minor": "3"
      }
    },
    "family": "Debian",
    "hardware": "x86_64",
    "name": "Debian",
    "release": {
      "full": "11.3",
      "major": "11",
      "minor": "3"
    },
    "selinux": {
      "enabled": false
    }
  }
}
root@basteles-bastelknecht ~ # puppet facts show lsbdistid
{
  "lsbdistid": "Debian"
}
root@basteles-bastelknecht ~ #
```